wallet/src/wallet.rs

```diff
@@ -122,6 +123,9 @@ pub trait CapeWalletExt<'a, Backend: CapeWalletBackend<'a> + Sync + 'a> {
         fee: u64,
     ) -> Result<TransactionReceipt<CapeLedger>, CapeWalletError>;
 
+    /// Get the ERC-20 asset code that this asset wraps, if this is a wrapped asset.
+    async fn wrapped_asset(&self, asset: AssetCode) -> Option<Erc20Code>;
```

Comment:
I think `wrapped_erc20` or `wrapped_erc20_code` is a better name since the "wrapped asset" should be the `AssetCode` itself and here we want to get the `Erc20Code` that it's wrapped from.

Reply:
Good point, will do